File: sample/dracula.swift
struct Vampire {
    let location: String
    let birthDate: Int
    let deathDate: Int
    let weaknesses: [String]

    var age: Int {
        deathDate - birthDate
    }
}

let dracula = Vampire(location: "Transylvania", birthDate: 1428, deathDate: 1476, weaknesses: ["Sunlight", "Garlic"])
let alucard = Vampire(location: "Wallachia", birthDate: 1458, deathDate: 1510, weaknesses: ["Holy Water"])

print("Dracula: \(dracula.age), Alucard: \(alucard.age)")

File: sample/dracula.py
from dataclasses import dataclass


@dataclass(frozen=True, slots=True)
class Vampire:
    location: str
    birth_date: int
    death_date: int
    weaknesses: tuple[str, ...]

    @property
    def age(self) -> int:
        return self.death_date - self.birth_date


dracula = Vampire("Transylvania", 1428, 1476, ("Sunlight", "Garlic"))
alucard = Vampire("Wallachia", 1458, 1510, ("Holy Water",))

print({"dracula_age": dracula.age, "alucard_age": alucard.age})